upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -346,6 +346,8 @@ func (c *ApplyClusterCmd) Run() error {
 	if err != nil {
 		return err
 	}
+	dnszone, _ := findZone(cluster, cloud) // any errors would be already caught in validateDNS
```

Comment:
Can we check the error - even if it shouldn't happen?  We do make calls to the cloud APIs, so I think it is possible that things fail the second time (e.g. rate limits).